Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -0,0 +1,124 @@
+## 195: Accessibility improvements in Windows Forms controls
+
+### Scope
+Major
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Windows Forms Framework is improving how it works with accessibility technologies to better support Windows Forms customers. These include the following changes:
+- Changes to improve display during High Contrast mode.
+- Changes to improve the property browser experience. Property browser improvements include: 
+  - Better keyboard navigation through the various drop-down selection windows.
+  - Reduced unnecessary tab stops.
+  - Better reporting of control types.
+  - Improved narrator behavior.
+- Changes to implement missing UI accessibility patterns in controls.
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+__How to opt in or out of these changes__
+  
+In order for the application to benefit from these changes, it must run on the .NET Framework 4.7.1 or later. The application can benefit from these changes in either of the following ways:
+- It is recompiled to target the .NET Framework 4.7.1.
+- It opts out of the legacy accessibility behaviors by adding the following [AppContext Switch](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the ```<runtime>``` section of the app config file and setting it to false, as the following example shows.
```

Comment:
URLs should not include "en-us" -- they'll fail to localize in that case. That's true for many of the URLs here.